services/device-registry-base/src/test/java/org/eclipse/hono/service/registration/AbstractRegistrationServiceTest.java

```diff
@@ -340,26 +343,23 @@ default void testCreateDeviceFailsForExistingDeviceId(final VertxTestContext ctx
         final String deviceId = randomDeviceId();
 
         createDevice(deviceId, new Device())
-            .onComplete(ctx.succeeding(ok -> {}))
```

Comment:
It would be in line with the other tests to keep this here, making sure the first device creation succeeded.